rllib/algorithms/pg/pg_tf_policy.py

```diff
@@ -49,14 +48,16 @@ def __init__(
             self,
             observation_space,
             action_space,
-            config,
+            config: PGConfig,
```

Comment:
Same for PG Policies: Demonstrate how future policies should utilize only `AlgorithmConfig`.

Note that all policies (other than Algorithm, RolloutWorker, WorkerSet) still receive old-style python dicts as configs. This will need to be cleaned up in a later step. Reasons for leaving this as-is for now:
* Some Policies do NOT have their own AlgorithmConfig classes, e.g. RandomPolicy, MockPolicy, yet do take some of their own specific config keys (str).
* Do not blow up this PR too much.